### PR TITLE
chore: archival notice for this repository

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # outline-go-tun2socks
 
+> [!WARNING]  
+> This repository is no longer being maintained. The tun2socks source is now maintained in the [outline-apps repository](https://github.com/Jigsaw-Code/outline-apps/tree/master).
```
Comment (fortuna) [inline README.md]: Maybe point to the tun2socks code?

**Review by fortuna — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jyyi1) [inline README.md]:
```suggestion
> This repository is no longer being maintained. The tun2socks source is now maintained in the [outline-apps repository](https://github.com/Jigsaw-Code/outline-apps/tree/master) and [Intra repository](https://github.com/Jigsaw-Code/Intra/tree/master/Android/app/src/go).
```

**Review by jyyi1 — COMMENTED** [review]
Suggest to mention Intra as well.

File: README.md (hunk 1 above)
Comment (daniellacosse) [reply README.md]: I'm about to move that code, and we can't edit this repository once it's archived, so I figured a pointer to the root of each repository was more stable?

**Review by jyyi1 — APPROVED** [review]

**Comment by jyyi1** [comment]
LGTM, we can just ignore the build errors.

File: README.md (hunk 1 above)
Comment (fortuna) [reply README.md]: Also the network package in the SDk!

**Pull request merged**